media-plugins/distrho-ports/distrho-ports-9999.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="+alsa +jack"
+
+RDEPEND="alsa? ( media-libs/alsa-lib )
+        jack? ( virtual/jack )"
```

Comment:
I deliberately left these out because it doesn't make sense to me that anyone would even bother installing this kind of software unless they already have a desktop environment already set up.